test/ts/migrations/3_deploy_test_contracts_test.ts

```diff
@@ -42,4 +46,77 @@ contract("Migration #3: Deploying Test Contracts", async (ACCOUNTS) => {
     let mockTokenTransferProxy: MockTokenTransferProxyContract;
     let mintableNonFungibleToken: MintableNonFungibleTokenContract;
     let tokenRegistry: TokenRegistryContract;
+
+    describe("Deployment", () => {
+        it("should deploy the `DummyContract` contract to the current network", async () => {
+            dummyContract = await DummyContractContract.deployed(web3, TX_DEFAULTS);
+            const doesContractExist = await web3Utils.doesContractExistAtAddressAsync(
```

Comment:
Nit: I think you forgot to change this to the `.eventually.be.true` pattern as well.  Please fix for consistency

Reply:
Good catch. This has been fixed.